### ISSUE TITLE
[DOCS] create a tutorial on using `TRL` and `RLHF` (via reward modeling and PPO)

### ISSUE THREAD

**Issue by davidberenstein1957** [recognai/rubrix#3560]
## Which page or section is this issue related to?
It might a nice use case to be able to use the `ArgillaTrainer()` for `PPO` and `trl` for showcasing how we might use the `FeedbackDataset` to gather bias and reduce bias/harmfullness from models.

**Comment by tomaarsen** [comment]
Superseded by the Mistral tutorial from #3889
